### PR TITLE
Fix search

### LINKED ISSUE #408: search text touches the search button
screen shot 2015-10-05 at 17 45 46

### PR DESCRIPTION
**Pull request by aju**
closes #408
screen shot 2015-10-07 at 09 35 05
screen shot 2015-10-07 at 09 35 18

### CODE REVIEW

**Comment by kdzwinel** [comment]
are you sure the button is not to close to the right edge of the input now?

**Comment by aju** [comment]
@JoannaWalusiak please take a look at screenshot and decide it it looks good

**Comment by JoannaWalusiak** [comment]
It doesn't look good, first active one has icon with too small padding from right,
second, inactive has too big icon.

**Comment by kdzwinel** [comment]
Also, please update the header example as discussed. I'll use it in the web client.

**Comment by aju** [comment]
code and screenshot updated

**Comment by vergilius** [comment]
:+1:

**Comment by aju** [comment]
@kdzwinel @vergilius @JoannaWalusiak please verify now

**Comment by JoannaWalusiak** [comment]
:+1: "daje okejkę"

File: src/components/search/_search.scss [hunk 1]
```diff
@@ -12,10 +12,13 @@ $includeHtml: false !default;
...
       top: 0;
       bottom: 0;
-      right: 6px;
+      right: gutter(0.5);
       display: flex;
       align-items: center;
-      padding-right: gutter(0.25);
+    }
+
+    &--short &__button {
```
Comment (kdzwinel) [inline src/components/search/_search.scss]: input that we put into a small search box is called mint-input--low, why this is `--short`? I think it should also be `--low`, WDYT?

Comment (aju) [inline src/components/search/_search.scss]: I think input has a wrong name:
https://github.com/brainly/style-guide/blob/master/CONTRIBUTING.md#modifier-dictionary

Comment (kdzwinel) [inline src/components/search/_search.scss]: good point, I forgot that we haven't fixed it. Reported - https://github.com/brainly/style-guide/issues/416

**Comment by kdzwinel** [comment]
ok

**Comment by vergilius** [comment]
:+1:

**Pull request merged**